app/src/main/res/values/strings.xml

```diff
@@ -272,7 +272,7 @@
     <!-- Home onboarding dialog welcome screen description text. -->
     <string name="onboarding_home_welcome_description">More colors. Better privacy. Same commitment to people over profits.</string>
     <!-- Home onboarding dialog sign into sync screen title text. -->
-    <string name="onboarding_home_sync_title">Switching screens is easier than ever</string>
+    <string name="onboarding_home_sync_title">Hop from phone to laptop and back</string>
```

Comment:
This doesn't need a `_2` because it was landed in the current cycle 2 days ago.

Reply:
That is incorrect. This needs a new ID because it was already exposed for localization (which happens multiple times in a cycle).